conf/timeseries-dev-source.conf

```diff
@@ -72,6 +72,11 @@
         # Thats about 6MB
         evicted-pk-bloom-filter-capacity = 50000
 
+        # Uncomment to log at DEBUG ingested samples matching certain filters
+        # Only works for StringColumn fields in the partition key
+        # trace-filters = {
+        #   metric = "bad-metric-to-log"
```

Comment:
Should we include _ns in the filter as well?
If the filter needs to be on a popular metric name, too much debug  can happen.

Reply:
For the current debug case, `StringColumn` filtering is fine, we don't need filtering by Map/labels.  We can extend it later if that is needed.  Let's keep it simple.